dotnet/dynamodb/README.md

```diff
@@ -0,0 +1,437 @@
+
+To create a global secondary index (GSI) for the three queries we made in
+*A simple example*,
+execute the following commands:
+
+```
+CreateIndex.exe -i OrderDate       -m Area -k string -s Order_Date       -t number
+CreateIndex.exe -i ProductOrdered  -m Area -k string -s Order_Product    -t number
+CreateIndex.exe -i LowProduct      -m Area -k string -s Product_Quantity -t number
+```
+
+Note that you cannot execute these commands one right after another.
+You must wait until one GSI is created before you can attempt to create another GSI.
+We recommend you use the Amazon DynamoDB console to monitor the progress
+of creating a GSI to avoid errors.
+
+## Reading data from a table
+
+You can read data from an Amazon DynamoDB table using a number of techniques.
+
+- By the item's primary key
+- By searching for a particular item or items based on the value of one or more keys
+
+## Modifying a table item
+
+Use the **UpdateItem** project to modify the status of an order in the table.
+
+The default table name is defined as **Table**,
+and the default region is defined as **Region**
+in *app.config*.
+
+It takes the following options:
+
+- ```-i``` *ID*, where *ID* is the value of the order's ORDER_ID attribute
+- ```-s``` *STATUS*, where *STATUS* is the new status value (backordered, delivered, delivering, or pending)
+
+This code example demonstrates two different levels of accessing an item in a table.
+- If you set the status (```-s``` *STATUS*) to **pending**,
+  the code example uses the **DynamoDBContext** class to load the table.
+  If the item is not an order, the update silently fails.
+- If you set the status to any other valid value,
+  the code example uses the lower-level **UpdateItemAsync** method,
+  which throws an exception if the item is not an order.
+
+## Deleting an item from a table
+
+Use the **DeleteItem** project to delete an item from the table.
+
+The default table name is defined as **Table**,
+and the default region is defined as **Region**
+in *app.config*.
+
+It takes the following option:
+
+- ```-p``` *PARTITION*, where *PARTITION* is the value of the partition key
+- ```-s``` *AREA*, where *AREA* is **Customer**, **Order**, or **Product**.
+
+If you provide an *AREA* value that does not match that of the item,
+the example silently fails to delete the item from the table.
+
+## Deleting items from a table
+
+Use the **** project to delete an item from the table.
+
+The default table name is defined as **Table**,
+and the default region is defined as **Region**
+in *app.config*.
+
+- ```-a``` *AREA*, where *AREA* is **Customer**, **Order**, or **Product**.
+- ```-i``` *IDS*, where *IDS* is a list of ID values, separated by spaces; all ID values must be for the associated *AREA*.
+
+## Deleting a table
+
+Use the **DeleteTable** project to delete a table.
+
+The default table name is defined as **Table**,
+and the default region is defined as **Region**
```

Comment:
default **Region** (and use bold on the file name

Reply:
Fixed